src/drive/web/modules/public/LightFileViewer.jsx

```diff
@@ -15,4 +17,8 @@ const LightFileViewer = ({ files }) => (
   </div>
 )
 
+LightFileViewer.propTypes = {
+  files: PropTypes.array.isRequired,
```

Comment:
could you use `arrayOf` so that we know exactly what do the files look like here?

Reply:
Since `files` is something coming from the API, it can have a lot of different shapes. 

Is it possible to only check for the presence of one or two keys ?